analysis_driver/quality_control/contamination_blast.py

```diff
@@ -1,40 +1,45 @@
-import os.path
 import json
+import os.path
 from ete3 import NCBITaxa
 from collections import Counter
+from luigi import Parameter, IntParameter
 from egcg_core import executor
-from .quality_control_base import QualityControl
 from analysis_driver.config import default as cfg
 from analysis_driver.exceptions import AnalysisDriverError
+from analysis_driver.segmentation import Stage
 
 
-class ContaminationBlast(QualityControl):
-    def __init__(self, dataset, working_dir, fastq_file):
-        super().__init__(dataset, working_dir)
-        self.fastq_file = fastq_file
-        self._ncbi = None
+class ContaminationBlast(Stage):
+    fastq_file = Parameter()
+    nb_reads = IntParameter(default=3000)
```

Comment:
`nb_reads` used to be hardcoded, but we can now override it if we like. Also removed passing round of tuple return values, as with Relatedness